### PR TITLE
added dry air medium for #824

### PR DESCRIPTION
**Pull request by Mathadon**
for #824 
@icupeiro see DryAir model
@damienpicard can you review this? This air medium does not contain moisture, such that it is linear.

### CODE REVIEW

File: IDEAS/Media/Specialized/BaseClasses/PartialSimpleMedium/package.mo [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+  redeclare function extends molarMass "Return the molar mass of the medium"
+  algorithm
+    MM := MM_const;
+  end molarMass;
+
+  annotation (Documentation(info="<html>
+<p>
+Partial medium that contains an explicit expression 
+for the computation of T as a function of u.
```
Comment (damienpicard) [inline IDEAS/Media/Specialized/BaseClasses/PartialSimpleMedium/package.mo]: mention the equation you have changed.

File: IDEAS/Media/Specialized/DryAir/package.mo [hunk 2]
```diff
@@ -0,0 +1,73 @@
+within IDEAS.Media.Specialized;
+package DryAir "Package with moist air model that decouples pressure and temperature"
```
Comment (damienpicard) [inline IDEAS/Media/Specialized/DryAir/package.mo]: adapt documentation string

File: IDEAS/Media/Specialized/DryAir/package.mo [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  extends IDEAS.Media.Specialized.BaseClasses.PartialSimpleMedium(
+     mediumName="Air",
```
Comment (damienpicard) [inline IDEAS/Media/Specialized/DryAir/package.mo]: check whether values are the same as in the Buildings library

File: IDEAS/Media/Specialized/BaseClasses/PartialSimpleMedium/package.mo [hunk 4]
```diff
@@ -0,0 +1,315 @@
+within IDEAS.Media.Specialized.BaseClasses;
+partial package PartialSimpleMedium "Medium model with linear dependency of u, h from temperature. All other quantities are constant, including density."
```
Comment (damienpicard) [inline IDEAS/Media/Specialized/BaseClasses/PartialSimpleMedium/package.mo]: mention that this media is used for linearization / optimization

**Comment by damienpicard** [comment]
@Mathadon this can be merged.

File: IDEAS/Media/Specialized/DryAir/package.mo (hunk 3 above)
Comment (Mathadon) [reply IDEAS/Media/Specialized/DryAir/package.mo]: there were indeed some errors here. See also https://github.com/Mathadon/modelica_MPC/commit/827601ca01b47bb6422383fd4568148e5d3718c8

**Pull request merged**